### ISSUE TITLE
TestFloat fails on fma tests

### ISSUE THREAD

**Issue by davidharrishmc** [openhwgroup/cvw#578]
sim-testfloat-batch all passes, but sim-testfloat-batch fma fails right away.
Why does one pass and the other fail, and which is right?

```
# Running f64_mulAdd_rne.tv vectors 
# 
# Error in f64_mulAdd_rne.tv
# TestNum          0 OpCtrl 0
# inputs: bcafffffe7ffffff 3fdfffffffffffff 3ff0000000000000
# SrcA: xxxxxxxxxxxxxxxx
#  Res: 3ff0000000000000 01
#  Expected: 3fefffffffffffff 01
```

**Comment by davidharrishmc** [comment]
This is a small negative number + 4.  It should be one ulp less than 4.  killprod is incorrectly asserted, wiping out the ulp and giving 4 as the answer instead.

**Comment by davidharrishmc** [comment]
@stineje Looks like all mode still doesn't check failures.

**Comment by davidharrishmc** [comment]
In the new test bench, these can be replicated by running
regression-wally -testfloat
Many _fma tests will fail for this reason.

**Comment by davidharrishmc** [comment]
Fixed by PR #793
